src/Messages.ts

```diff
@@ -306,9 +306,31 @@ export default class Messages {
     return `rgba(${r},${g},${b},${a})`
   }
 
+  private addDefaults(vars: MessageVariables): MessageVariables {
+    const kinds = this.getMessages().actionDefinitions || {}
+    const defaults = kinds[vars.__name__]
+
+    if (!defaults) {
+      return vars
+    }
+
+    function useDefaults(obj: MessageVariables, defaultValues: MessageVariables): MessageVariables {
+      for (const key in defaultValues) {
+        const value = defaultValues[key]
+        if (typeof value === 'object') {
+          obj[key] = useDefaults(obj[key] || {}, value)
+        } else if (!obj[key]) {
```

Comment:
This probably should be changed otherwise defaults might override valid values which evaluate to `false`:
```suggestion
        } else if (obj[key] === undefined) {
```

Reply:
Thanks for noticing this! 💯